x-pack/plugin/runtime-fields/build.gradle

```diff
@@ -15,6 +15,8 @@ tasks.withType(JavaCompile).configureEach {
 dependencies {
   compileOnly project(":server")
   compileOnly project(':modules:lang-painless:spi')
+  api project(':libs:elasticsearch-grok')
+  api project(':libs:elasticsearch-dissect')
```

Comment:
I'm not sure I ever know the difference between `api` and `compileOnly`. I just copied these two lines from `ingest-common`.